composeApp/src/commonMain/kotlin/com/sixbynine/transit/path/widget/WidgetDataFetcher.kt

```diff
@@ -109,6 +113,7 @@ object WidgetDataFetcher {
             isPathError: Boolean,
             data: DepartureBoardData?,
         ) -> Unit,
+        isCommuteWidget: Boolean = false,
```

Comment:
per our discussion, I'd probably remove the default value here.

It's a little annoying to pass the explicit value in more places, but really annoying to forget to propagate it somewhere and cause a bug